bls-signatures/benches/bls_signatures.rs

```diff
@@ -0,0 +1,93 @@
+use {
+    criterion::{black_box, criterion_group, criterion_main, Criterion},
```

Comment:
nit: `black_box` from criterion has been deprecated in favor of `std::hint::black_box` -- maybe that should be used instead?

https://docs.rs/criterion/latest/criterion/fn.black_box.html

Reply:
Good point. Updated!